app/src/test/java/com/pokerio/app/GameStateTest.kt

```diff
@@ -83,5 +129,180 @@ class GameStateTest {
         assertTrue("smallBlind not reset", GameState.smallBlind == -1)
         assertTrue("isPlayerAdmin not reset", GameState.isPlayerAdmin == false)
```

Comment:
Can you change it to `!GameState.isPlayerAdmin` or simply change `assertTrue` to `assertFalse`?

Reply:
Done